package.json

```diff
@@ -92,7 +92,20 @@
   },
   "resolutions": {
     "@apollo/client": "3.7.4",
+    "@folio/circulation": "7.1.0",
+    "@folio/plugin-create-inventory-records": "3.1.0",
+    "@folio/plugin-find-contact": "3.2.0",
+    "@folio/plugin-find-eresource": "4.2.0",
+    "@folio/plugin-find-fund": "1.1.0",
+    "@folio/plugin-find-instance": "6.2.0",
+    "@folio/plugin-find-interface": "3.2.0",
+    "@folio/plugin-find-organization": "3.2.0",
+    "@folio/plugin-find-po-line": "3.2.0",
+    "@folio/plugin-find-user": "6.3.0",
+    "@folio/quick-marc": "5.1.3",
     "@folio/react-intl-safe-html": "3.1.0",
+    "@folio/stripes-acq-components": "3.2.4",
+    "@folio/stripes-data-transfer-components": "5.3.0",
```

Comment:
I think this should be `5.2.1`; `5.3.0` was the Nolana/2022-R3 release. `5.3.0` may be safe, but I don't know if it was explicitly tested together.

Reply:
right.  I figured 5.3.0 was safe because it looked to be just translations updates,  but I'll change it to 5.2.1.